src/main/resources/anniv5Resources/localization/eng/siegepack/Relicstrings.json

```diff
@@ -12,7 +12,7 @@
     "FLAVOR": "Recycles leftover energy into manufacturing prowess!",
     "DESCRIPTIONS": [
       "Consuming a #y${ModID}:Shell increases their effect by #b",
-      " this combat."
+      " this combat. Whenever you use a potion, gain a #y${ModID}:Shell."
```

Comment:
If you feel the effect is too weak then maybe increase the base effect to 2 instead? Adding separate effects on top is not a great way to go about it.